README.md

```diff
@@ -289,7 +289,10 @@ To find information about supported submodules and there version in SDE please v
 | **GET:- localhost:8080/api/legal-entities** |This API is used to fetch legal entities (list of company's) for Process| Refer Api Doc |Refer Api Doc|
 | **POST:- localhost:8080/api/connectors-discovery** |This API is used to fetch connectors information | Refer Api Doc |Refer Api Doc|
 
+- Detailed API specs available under:
 
+https://github.com/catenax-ng/tx-managed-simple-data-exchanger-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml
```

Comment:
wouldn't be better to use relative links here? This PR is targeting tractus-x repo, so link becomes external.

Reply:
Corrected